file.go

```diff
@@ -1,37 +1,127 @@
 package jcsv
 
-import "os"
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	jsonData []uint8
+	csvData [][]string
+	csvHeader []string
 }
 
 func ParseJsonFile(path string) (file, error) {
-	// TODO: open and read the given file into your `file` object
-	return file{}, nil
+	var fileObj file
+	fileStream,fileOpenError := os.Open(path)
+	if fileOpenError!=nil{
+		return file{},fileOpenError
+	}
+	var fileReadError error
+	fileObj.jsonData,fileReadError=ioutil.ReadAll(fileStream)
```

Comment:
Does assignment to `fileObj.jsonData` works?
Have you tested it for whether it causes a null pointer dereference error or not?